audio_play/src/audio_play.cpp

```diff
@@ -16,14 +16,16 @@ namespace audio_transport
 
         std::string dst_type;
         std::string device;
+        std::string topic;
         bool do_timestamp;
 
         // The destination of the audio
         ros::param::param<std::string>("~dst", dst_type, "alsasink");
         ros::param::param<std::string>("~device", device, std::string());
         ros::param::param<bool>("~do_timestamp", do_timestamp, true);
+        ros::param::param<std::string>("~topic", topic, std::string());
```

Comment:
Why use `std::string()` as the default value (if that's actually it)? Maybe we could set it as `audio` like how it worked before?

Reply:
I copy pasted the device param and changed it to topic

Reply:
Please see this [comment](https://github.com/clubcapra/capra_audio_common/pull/2/files/d5899c5a01fb82e66ec17092aa8b643797dc6de0#r625395389)